py_gtfs_rt_ingestion/pyproject.toml

```diff
@@ -14,6 +14,32 @@ boto3 = "^1.23.3"
 pytest = "^5.2"
 pandas = "^1.4.2"
 pytest-cov = "^3.0.0"
+black = "^22.3.0"
+pylint = "^2.14.1"
+mypy = "^0.961"
+
+[tool.black]
+line-length = 80
+target-version = ['py39']
+
+[tool.mypy]
+python_version = 3.9
+ignore_missing_imports = true
+disallow_untyped_defs = true
+warn_unused_ignores = true
+warn_unreachable = true
+pretty = true
+
+[tool.pylint]
+disable = [
+  # disable doc string requirements
+  "missing-module-docstring",
+  # allow catching a generic exception
+  "broad-except"
+]
+good-names = ["e"]
+max-line-length = 80
+min-similarity-lines = 10
```

Comment:
these are basic configurations i arrived at. if you have anything in here you would want to change lmk.